src/transaction/tests/test_weakset.py

```diff
@@ -137,4 +137,4 @@ class Dummy:
 
 
 def test_suite():
-    return unittest.makeSuite(WeakSetTests)
+    return unittest.TestLoader().loadTestsFromTestCase(WeakSetTests)
```

Comment:
This is going to be fixed in #116 by `pyupgrade`.

Reply:
Rebased out!